source/jormungandr/jormungandr/instance_manager.py

```diff
@@ -232,19 +234,51 @@ def _find_coverage_by_object_id_in_instances(self, instances, object_id):
                 raise InvalidArguments(object_id)
             return self._all_keys_of_coord_in_instances(instances, flon, flat)
 
+        # Request without coverage and pt_object id (from or to)
         return self._all_keys_of_id_in_instances(instances, object_id)
 
     def _all_keys_of_id_in_instances(self, instances, object_id):
-        valid_instances = []
-        for instance in instances:
-            if self._exists_id_in_instance(instance.name, instance.publication_date, object_id):
-                valid_instances.append(instance)
-        if not valid_instances:
-            raise RegionNotFound(object_id=object_id)
+        # Get the first occurrence pt_object coordinate and manage as above
+        # If no object with coordinate exist among all the authorized instances
+        # Then we will be obliged to call all krakens as before (necessary for test with bad data)
+        object_coord = self._get_first_object_coord_in_instances_by_id(instances, object_id)
+        if object_coord:
+            return self._all_keys_of_coord_in_instances(instances, object_coord.lon, object_coord.lat)
```

Comment:
:rotating_light: To me, it looks like 2 problems are gonna rise:
1. Candidate coverages will sometimes not include coverages that know the stop-id
  - Ex. A: A journey without coverage from Lille to Aix (for a free-user of Navitia).
    Candidates are `fr-ne` only for Lille, `fr-se` only for Aix: Navitia won't be able to find a single coverage matching the journey (it is something we are currently able to do).
  - Ex. B: A journey without coverage from Nantes to Lisboa.
    No candidate (from coords) for Lisboa (no `pt` coverage), so no journey, whereas `fr-nw` is able to process a journey.
2. Candidate coverages will sometimes include coverages that don't know the stop-id (they just have coords in their shape)
  - Ex: A journey for a "full-access" user from LilleFlandres (stop-id from `fr-ne`) to LilleEurope (stop-id from `fr-ne` also):
    From the priorities, `fr-npdc` will be used instead of `fr-ne`, and `fr-npdc` doesn't know the stop-ids from `fr-ne`.

As discussed: not sure about the best way to progress considering that (@stifoon may have ideas).
* Really replacing by stop-coords the stop-ids (probably with a free_radius for the case of Lisboa) when using no coverage
* Having multiple types of candidates (they may be implicit to save some performance):
  - golden: the ones that know both stop-id and shape includes stop-coord
  - silver: the ones that just know stop-id but shape doesn't include stop-coord
  - bronze: the ones that don't know stop-id, but shape includes stop-coord (this one is probably just a nice-to-have as it would improve current feature and requires switching on stop-coord everywhere in the call
* Switch to mimir if it does know both coverages and coords for a given stop-id